doc/source/whatsnew/v1.3.0.rst

```diff
@@ -877,7 +877,7 @@ Sparse
 
 - Bug in :meth:`DataFrame.sparse.to_coo` raising ``KeyError`` with columns that are a numeric :class:`Index` without a 0 (:issue:`18414`)
 - Bug in :meth:`SparseArray.astype` with ``copy=False`` producing incorrect results when going from integer dtype to floating dtype (:issue:`34456`)
--
+- Bug in :class:`SparseArray` type as :meth:`max` and :meth:`min` do not exist (:issue:`40921`)
```

Comment:
suggestion: "Implemented :meth:\`SparseArray.min\` and :meth:\`SparseArray.max\` (:issue:\`40921\`)"